dist/rpm/openQA.spec

```diff
@@ -650,6 +652,8 @@ fi
 %files bootstrap
 %{_datadir}/openqa/script/openqa-bootstrap
 %{_datadir}/openqa/script/openqa-bootstrap-container
+%{_bindir}/openqa-bootstrap
+%{_bindir}/openqa-bootstrap-container
```

Comment:
why this change? Has it anything to do with the other commit? I think we decided that we don't want the script to be within the main bin dir so that users do not trigger it by accident or "curiousity" exploring options of tab-completion :)